app/core/ingestion/utils.py

```diff
@@ -103,6 +103,14 @@ class IngestContext:
     results: list[Result]
 
 
+@dataclass
+class ValidationResult:
```

Comment:
New Response object for the validation endpoint

Reply:
Potentially a boolean of whether valid or not?